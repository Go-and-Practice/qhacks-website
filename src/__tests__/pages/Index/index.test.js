import React from "react";
import IndexPage from "../../../pages/index";
import { shallow } from "enzyme";

describe("IndexPage", () => {
  it("renders correctly", () => {
    expect(shallow(<IndexPage />)).toMatchSnapshot();
  });
});
