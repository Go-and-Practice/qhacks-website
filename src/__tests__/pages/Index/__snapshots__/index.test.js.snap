// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders correctly 1`] = `
<div>
  <section
    id="headerLanding"
  >
    <div
      id="headerTitle"
    >
      <img
        className="logo"
        src="test-file-stub"
      />
      <h2
        className="subTitle"
      >
        Queen's University | Winter 2019
      </h2>
      <div
        className="btnRow"
      >
        <a
          className="btn btnMain yellowBtn"
          href="https://2018.qhacks.io"
        >
          QHacks 2018
        </a>
        <a
          className="btn btnMain orangeBtn"
          href="mailto:partnership@qhacks.io"
        >
          Partner
        </a>
      </div>
      <div
        className="talkToUs"
      >
        Talk to us at:
         
        <a
          className="frontContactLink"
          href="mailto:hello@qhacks.io"
        >
          hello@qhacks.io
        </a>
      </div>
      <div
        className="talkToUs"
      >
        This year's website is under construction!
      </div>
    </div>
  </section>
</div>
`;
